lib/ansible/plugins/action/win_reboot.py

```diff
@@ -89,4 +91,12 @@ def perform_reboot(self):
         except AnsibleError:
             display.debug("%s: connect_timeout connection option has not been set" % self._task.action)
 
+        post_reboot_delay = int(self._task.args.get('post_reboot_delay', self._task.args.get('post_reboot_delay_sec', self.DEFAULT_POST_REBOOT_DELAY)))
```

Comment:
Should this be shared in the `run` method after `perform_reboot` is called? That way the same code is used for win_reboot and reboot.

Reply:
That's a good suggestion. I have updated the code like this.